packages/manager/apps/public-cloud/package.json

```diff
@@ -8,9 +8,9 @@
     "build": "webpack --env.production",
     "dev": "webpack-dev-server",
     "dev:watch": "yarn run dev",
-    "start": "lerna exec --stream --scope='@ovh-ux/manager-public-cloud' --include-filtered-dependencies -- yarn run build",
-    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-public-cloud' --include-filtered-dependencies -- yarn run dev",
-    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-public-cloud' --include-filtered-dependencies -- yarn run dev:watch"
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-public-cloud' --include-filtered-dependencies -- npm run build --if-present",
```

Comment:
Is there any benefit to make a distinction between `yarn` and `npm` in that case?

Reply:
`yarn` doesn't have a `--if-present` option

Reply:
Noticed!

But in my point of view I think we should keep all theses scripts. In that way it will ensure, as a module level, that we can keep building it and figured out what's going otherwise it will be delegate to the Public Cloud webpack config which can increase the building time for this given module.